    syntax = "proto3";

    package audio_spoof; // Оставляем ваш пакет для совместимости импортов

    service AudioDetection { // Сервис как в README и новом grpc_server.py
      rpc ProcessAudio (AudioDataRequest) returns (AudioDetectionResponse);
    }

    message AudioDataRequest {
      string request_id = 1;
      string minio_object_key = 2; // Ключ объекта (путь к файлу) в MinIO
      string original_filename = 3; // Для информации и возможного выбора кодека
    }

    message ChunkPrediction { // <- Вот недостающее сообщение
      // Оценка вероятности того, что чанк является дипфейком (0.0 до 1.0).
      // 0.0 означает "очень вероятно настоящий", 1.0 - "очень вероятно фейк".
      float score = 1;
    }

    message AudioDetectionResponse { // <- Вот недостающее сообщение
      string request_id = 1;
      map<string, ChunkPrediction> chunk_predictions = 2; // e.g., "chunk_0", "chunk_1"
      string error_message = 3; // Если была ошибка в Python сервисе
    }